docs/application/dotnet/guides/multimedia/face-recognition.md

```diff
@@ -0,0 +1,76 @@
+# Deep Learning Based Face Recognition
+
+Using deep learning based face recognition, you can perceive and understand faces within your application. This document is a guide on how to register a new face, unregister a face that is registered with a given label, and recognize a given face.
+
+## Prerequisites
+
+To enable your application for using the deep learning based face recognition functionality, create `DeepLearningFaceREcognizer` instance.
```

Comment:
I think there has been a spelling mistake here. Also, it would be better if the DeepLearningFaceRecognizer class was linked (https://docs1.stg.tizen.org/application/dotnet/api/TizenFX/latest/api/Tizen.Multimedia.Vision.DeepLearningFaceRecognizer.html) instead of placing it in a code tag.

```suggestion
To enable your application for using the deep learning based face recognition functionality, create `DeepLearningFaceRecognizer` instance.
```